graylog2-server/src/main/java/org/graylog2/plugin/inject/Graylog2Module.java

```diff
@@ -52,6 +52,7 @@
 import org.graylog2.plugin.outputs.MessageOutput;
 import org.graylog2.plugin.security.PasswordAlgorithm;
 import org.graylog2.plugin.security.PluginPermissions;
+import org.graylog2.web.PluginUISettingsProvider;
```

Comment:
Unrelated change?

Reply:
Yes, that was a leftover from a previous version.  Will clean up.